casper/message.py

```diff
@@ -19,13 +19,37 @@ def __hash__(self):
         # use of __hash__ in dictionaries, sets, etc
         return hash(self.hash)
 
-    def __eq__(self, message):
+    @staticmethod
+    def is_message(message):
```

Comment:
Because we use the `if message is None` + `isinstance` a ton of times with all of the comp methods. Just realized though that we don't need to check for `None`... `isinstance(None, Message)` returns false so... fixing to just have `isinstance`